assets/sass/additionalConditions.scss

```diff
@@ -84,12 +84,21 @@
   }
 }
 
+%container-shared {
```

Comment:
Maybe these should be moved to application.scss as it's no longer just used in additional conditions